python/cuspatial/cuspatial/core/gis.py

```diff
@@ -335,3 +338,40 @@ def polyline_bounding_boxes(poly_offsets, xs, ys, expansion_radius):
     return DataFrame._from_data(
         *cpp_polyline_bounding_boxes(poly_offsets, xs, ys, expansion_radius)
     )
+
+
+def pairwise_linestring_distance(xs1, ys1, offsets1, xs2, ys2, offsets2):
+    """Compute the distances of pairs of linestrings from two arrays.
```

Comment:
Let's use the same description as in the C++ API.

```suggestion
    """Compute shortest distance between pairs of linestrings (a.k.a. polylines).
```

Please also copy the examples from the C++ API here.